client/src/app/shared/header/header.component.ts

```diff
@@ -1,15 +1,35 @@
 import { Component, OnInit } from '@angular/core';
+import { AppRoutes } from '../../app-routes.module';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.css']
 })
 export class HeaderComponent implements OnInit {
+  routes: Array<any> = AppRoutes;
```

Comment:
Don't you need to declare with const or let for this routes variable?

Reply:
For class variables, you don't need to. Actually, when you use const or let, it will give you an error. That's typescript syntax.
https://www.tutorialspoint.com/typescript/typescript_classes.htm

Reply:
cool. thx